hr_timesheet_portal/templates/portal.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2021 Hunki Enterprises BV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <template id="portal_my_task" inherit_id="hr_timesheet.portal_my_task">
```

Comment:
Would be cool to add a sorting/grouping header here as well, but optional

Reply:
let's leave that to version 2 if there's not just some class we can slap on the table for that?

Reply:
It's a bit [more complicated](https://github.com/OCA/OCB/blob/12.0/addons/sale/controllers/portal.py#L87), although still fairly easy I think. Anyway lets leave it